docs/book/src/component-config-tutorial/testdata/project/Makefile

```diff
@@ -118,15 +118,15 @@ CONTROLLER_TOOLS_VERSION ?= v0.9.0
 KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
```

Comment:
About the title could you please rename this one for : 

`:bug: (go/v3): fix makefile targets do not re-install binaries when they are found in the project bin directory` 

Note that we use the title to generate the release notes.

Reply:
Also, could you please improve the description of the PR with something like:

**Description**

Change the Makefile targets controller-gen and envtest no longer try to download the binary when they can be found in the project/bin directory.  

**Motivation**

Closes: idIssue